Deployment_files/3_database-deployment.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: roboto-postgres
+  namespace: discord-bot-py
+  labels:
+    app: roboto-postgres
+spec:
+  replicas: 1
```

Comment:
where are the requests and cpu limits at

Reply:
I'm taking this up at the next CAB advisory board meeting.
Thank you for your input!